CliFx/Domain/HelpTextWriter.cs

```diff
@@ -246,21 +246,45 @@ void RenderOptions()
 
                     RenderColumnIndent();
 
-                    // Description
-                    if (!string.IsNullOrWhiteSpace(option.Description))
-                    {
-                        Render(option.Description);
-                        Render(" ");
-                    }
+                    RenderOptionDescription(option);                    
+
+                    RenderNewLine();
+                }
+            }
+
+            void RenderOptionDescription(CommandOptionSchema option)
+            {
+                // Description
+                if (!string.IsNullOrWhiteSpace(option.Description))
+                {
+                    Render(option.Description);
 
-                    // Environment variable
-                    if (!string.IsNullOrWhiteSpace(option.EnvironmentVariableName))
+                    // Valid values for enum types.
+                    // Property can actually be null here due to damn it operator in CommandArgumentSchema.cs:103,106.
+                    var propertyType = option.Property?.PropertyType;
+
+                    if (propertyType is object && propertyType.IsEnum)
                     {
-                        Render($"(Environment variable: {option.EnvironmentVariableName}).");
-                        Render(" ");
+                        // Cast to object so that we can use linq methods on the Array.
+                        var values = Enum.GetValues(propertyType)
+                            .Cast<object>()
+                            .Select(v => v.ToString().ToLowerInvariant());
+
+                        // Only render valid values if the enum is not empty.
+                        var validValues = values.Any()
+                            ? $" Valid values: {string.Join(", ", values)}."
+                            : "";
```

Comment:
Again, I think we can just safely assume that the enum has values. I can't imagine a use case where an enum without values is used as option.

Reply:
In order to support getting the valid values of other types in the future, I left a similar check in order to not print `Valid Values: ` for unbounded valid values.

Reply:
Good point